src/common/vars.ts

```diff
@@ -5,7 +5,7 @@ import { defineGlobal } from "./utils/defineGlobal";
 
 export const isMac = process.platform === "darwin"
 export const isWindows = process.platform === "win32"
-export const isDebugging = process.env.DEBUG === "true";
+export const isDebugging = process.env.DEBUG === "lens";
```

Comment:
please revert this and let's focus on this in separate PR

Reply:
Done